libs/zedUpload/go.mod

```diff
@@ -18,3 +18,5 @@ require (
 )
 
 replace github.com/lf-edge/eve/api/go => ../../api/go
+
+replace github.com/lf-edge/eve/pkg/pillar => ../../pkg/pillar
```

Comment:
Why do we need this? The dependency is from pillar to zedUpload and here it looks like it is the other way around. Are you using some definitions from pillar?